pkg/promutil/prometheus.go

```diff
@@ -6,6 +6,7 @@ import (
 
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/common/model"
+	"golang.org/x/exp/maps"
```

Comment:
is there a needed difference in using this one vs https://pkg.go.dev/maps?

Reply:
The exp package version gives me a slice of the key values which is what I'm after. The iterator based version added to the standard lib in 1.23 isn't of much use to me vs the exp version.